src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -44,57 +46,58 @@
 
 ;; ACE creation
 
-(defn munge-name
-  [pattern]
-  (-> pattern (str/lower-case) (str/split #"\.") reverse vec))
+(schema/defn ^:private split-domain :- [String]
+  "Given a domain string, split it on '.' and reverse it. For examples,
+  'sylvia.plath.net' becomes ['net' 'plath' 'sylvia']. This is used for domain
+  matching."
+  [pattern :- String]
```

Comment:
Perhaps rename this ... I would expect a parameter named `pattern` to be a regex.  I *think* that's a convention used elsewhere, but I could be wrong.